guides.md

```diff
@@ -237,6 +239,140 @@ members. Skip the steps below at your discretion for low-stakes topics.
   1. Use the private dial-in number from our [technical Jitsi documentation](https://github.com/hyphacoop/organizing-private/blob/master/documents/infrastructure/jitsi.md).
   2. _After_ prompt, enter conference code `307 314 3734 #`
 
+## Onboarding Checklist
+
+This is a checklist for onboarding a new Member to the Co-operative 🚀
+
+### Setting up communications
+
+1. Get Signal number and add to group.
```

Comment:
Worth mentioning here that https://link.hypha.coop/employees should be populated as going through this?